### DISCUSSION TITLE
How to crosscompile fast-DDS?

### DISCUSSION THREAD

**Discussion by wjhtinger** [eProsima/Fast-DDS#3650] [Unofficial support]
### Is there an already existing issue for this?

- [X] I have searched the existing issues

### Expected behavior

I want to run fast-DDS on ARM soc, how to crosscompile? Where has the guide and example?

### Current behavior

.

### Steps to reproduce

.

### Fast DDS version/commit

.

### Platform/Architecture

Other. Please specify in Additional context section.

### Transport layer

Default configuration, UDPv4 & SHM

**Comment by JLBuenoLopez** [comment]
Thanks for your contribution @wjhtinger 

Crosscompiling is not officially supported, though being Fast DDS a CMake project you would need to define your [toolchain file](https://cmake.org/cmake/help/book/mastering-cmake/chapter/Cross%20Compiling%20With%20CMake.html). I am moving this issue to the proper forum according to [Fast DDS CONTRIBUTING guidelines](https://github.com/eProsima/Fast-DDS/blob/master/CONTRIBUTING.md#issues-and-support). In that forum there are some other discussions between community members about crosscompiling in several platforms that might be helpful for your use case.
